docs/docs-content/profiles/cluster-profiles/modify-cluster-profiles/update-cluster-profile.md

```diff
@@ -122,66 +230,64 @@ following steps will guide you in updating configurations.
 Ensure you follow these practices when updating to a new pack version.
 
 - You should not copy the pack configuration from one version to another, as the newer version often contains
-  customizations. Instead, you should integrate your changes manually in the new version.
+  customizations. Instead, you should integrate your changes manually in the new version. Use the **Keep** button to
+  copy the lines from the current configuration to the new version.
 - Update to a newer Kubernetes version incrementally, one minor version at a time.
 - Select a specific target version instead of a group that ends in `.x`
 - We do not recommend downgrading packs to the previous version.
 
 :::
 
+### Prerequisites
+
+- A cluster profile created in Palette.
+
+- There are updates available for at least one pack in the profile.
+
+- Your Palette account role must have the `clusterProfile.update` permission to update a profile. Refer to the
+  [Cluster Profile permissions](../../../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile-admin)
+  reference for more information about roles and permissions.
+
+### Update Pack Version
+
 1. Log in to [Palette](https://console.spectrocloud.com).
 
 2. From the left **Main Menu**, select **Profiles**.
 
-3. Select the profile you want to update. Palette displays profile details and the profile stack.
-
-4. Click on the pack layer to update.
+3. Select the profile you want to update to access the profile details page.
 
-5. In the **Edit Pack** panel, select a specific target version, not a group that ends in `.x`. Palette displays the
+4. In the **Edit Pack** panel, select a specific target version, not a group that ends in `.x`. Palette displays the
    difference between the current version at left and the new version at right. The target version is displayed in the
    header.
 
    Differences between the displayed configurations are as follows:
 
-   - _Red highlight_ indicates text that is not present in the new configuration.
-
-     These may be lines you have added in the current configuration. Use the arrow that displays between the two
-     configurations to transfer the lines to the new pack version.
+   - _Yellow highlight_ indicates text that is not present in the new configuration. These may be lines you have added
+     in the current configuration. These lines may also have been removed because they are no longer valid in the new
+     configuration. If you need them, you can use the **Keep** button to transfer the lines to the new pack version.
```

Comment:
```suggestion
     in the current configuration or lines that have been removed because they are no longer valid in the new
     configuration. If you need them, you can use the **Keep** button to transfer the lines to the new pack version.
```

maybe? :)